### DISCUSSION TITLE
ScaleIntensityRanged - Parameters

### DISCUSSION THREAD

**Discussion by agokli** [Project-MONAI/MONAI#8162] [Q&A]
I like to understand defining parameters for ScaleIntensityRanged:

(a) a_min (float) – intensity original range min
(b) a_max (float) – intensity original range max.

Question  - How do I define a_min and a_max for all of images?
Question - How do they relate to Window and Level?
Question - I am using 3D Slicer to get Window and Level. How do I use these to define a_min and a_max?

spleen_segmentation_3d example has above parameters set to:
a_min =-57
a_max=164

Need to understand how they are defined.

**Comment by KumoLiu** [comment]
Hi @agokli, thanks for your interest here.

In `ScaleIntensityRanged`, the parameters `a_min` and `a_max` represent the minimum and maximum values of the intensity range that the original image intensities will be mapped from. This is a linear rescaling of the pixel intensity values from the original range [a_min, a_max] to a new range, often [0, 1] or [0, 255] depending on the application.
You can use `DataAnalyzer` to automatically analyzes given medical image dataset and reports the statistics.
For the relationship between WW, WL and intensity range, you can refer to:https://radiopaedia.org/articles/windowing-ct

Thanks.
